docs/sources/http_api/preferences.md

```diff
@@ -66,6 +66,32 @@ Content-Type: text/plain; charset=utf-8
 {"message":"Preferences updated"}
 ```
 
+## Patch Current User Prefs
+
```

Comment:
Can we add a sentence - an intro/overview. It is OK if it is brief.

Reply:
i've added a short description, but it's worth mentioning that neither the `GET` or `PUT` requests currently have a description either, so `PATCH` is now the odd one out 😄